assets/webhook/configuration.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validation.csi.vsphere.vmware.com
+  annotations:
+    service.beta.openshift.io/inject-cabundle: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+webhooks:
+  - name: validation.csi.vsphere.vmware.com
+    clientConfig:
+      service:
+        name: vmware-vsphere-csi-driver-webhook-svc
+        namespace: openshift-cluster-csi-drivers
+        path: /validate
+    rules:
+      - apiGroups:   ["storage.k8s.io"]
+        apiVersions: ["v1", "v1beta1"]
+        operations:  ["CREATE", "UPDATE"]
+        resources:   ["storageclasses"]
+    sideEffects: None
+    admissionReviewVersions: ["v1"]
+    failurePolicy: Fail
```

Comment:
Currently, the static controller will update this object on every sync (xref: https://github.com/openshift/library-go/pull/1129#discussion_r670207427)

Reply:
The linked issue appears to be fixed? lets update library-go ?

Reply:
sorry looking at linked PR, I think underlying issue is still unsolved.